src/app/lib/config/services/getConfig/index.js

```diff
@@ -0,0 +1,22 @@
+import pathOr from 'ramda/src/pathOr';
+
+/* This function allows for a dynamic import of service config outside of
+the react application (for example, for server side data fetching). Within
+the react application, the config can be found in the ServiceContext. */
+const getConfig = async (service, variant = 'default') => {
```

Comment:
I'm  a little uncertain about this - but this seems like the best way of being able to use the service config outside of the react application, which appears to be impossible at the moment? Using a normal static import of all of the config and pulling out the one we need would bloat the bundle massively. This means that we're importing the config for this service here in one way, and using a loadable react context in the react application. This seems like a sensible option without a big refactor in the way be load config, but I'm very open to suggestions.

Reply:
What if we move this so it's not co-located with the config files, but only in the routes, where we need to access config like this?